POLICIES.md

```diff
@@ -40,9 +39,14 @@ References: [[1](https://github.com/exercism/java/issues/177#issuecomment-261291
 
 References: [[1](https://github.com/exercism/java/issues/178)], [[2](https://github.com/exercism/java/pull/683#discussion_r125506930)]
 
-### Do not make user-facing classes `final`
+### Avoid using final
 
-> Starter implementations and test classes should not include the `final` keyword in class declarations as it isn't strictly necessary and could be confusing to people unfamiliar with Java.
+> Avoid using `final` in user-facing code. Liberal use of the `final` keyword remains controversial. It adds little value to test suites, and if included in starter implementations, places implicit restrictions on user solutions.
+
+### Adhere to best practices
+
+> Ensure that all Java code adheres to the best practices listed below:
+> - minimize the accessibility of classes and members (Effective Java, item 13)
```

Comment:
Unfortunately there's no way to link to content of the book itself, as far as I know. I could link to some kind of proxy like http://jtechies.blogspot.com/2012/07/item-13-minimize-accessibility-of.html?